### DISCUSSION TITLE
Installation under Plesk

### DISCUSSION THREAD

**Discussion by sapientorius** [directus/directus#18307] [Archive]
### Subject Area

Self hosting

### Suggestions

Since a few versions it's getting harder to install Directus on Plesk instances.

I remember that there has been a tutorial in the documentation where it was explained how to do it, but it's gone and it doesn't work anymore.

Is there a preferred way for the current version how to install it on Plesk without using docker?

**Comment by rijkvanzanten** [comment]
> Is there a preferred way for the current version how to install it on Plesk without using docker?

The preferred way is... use Docker 👀 There have a lot of questions / false positive bug reports just because of incompatibilities in the environment caused by the usage of Plesk and/or a shared host that doesn't have the proper server dependencies available that we were forced to stop offering that as a core-supported docs page. The Docker image solves that, as it includes all the system dependencies node/directus relies on 👍🏻 

I don't think we'll be able to re-introduce and maintain the plesk docs in an official capacity

**Reply by sapientorius** [reply]
Thank you for clarification!

I mean it just runs smoothly on our Server with Plesk. It's just overcomplicated to install it :)

**Comment by sapientorius** [comment]
For the community:

the old solution has to be changed a little bit:

1. Add directus and database-package to package.json
2. Run npm install through Plesk interface
3. Create an index.cjs file in the root folder with this code:

```
async function main() {
  const { startServer } = await import('@directus/api/server');
  startServer();
}

main();
```

4. Set index.cjs as startup file in the NodeJS settings for your site
